client/network/src/service.rs

```diff
@@ -897,7 +912,11 @@ impl<B: BlockT + 'static, H: ExHashT> Future for NetworkWorker<B, H> {
 		// Check for new incoming light client requests.
 		if let Some(light_client_rqs) = this.light_client_rqs.as_mut() {
 			while let Poll::Ready(Some(rq)) = light_client_rqs.poll_next_unpin(cx) {
-				this.network_service.user_protocol_mut().add_light_client_request(rq);
+				// This can error if there are too many queued requests already.
+				let _ = this.network_service.light_client_request(rq);
```

Comment:
Should we not log a message here if an error has been returned? Are there plans to replace those unbounded channels with something that can exercise back pressure?

Reply:
> Are there plans to replace those unbounded channels with something that can exercise back pressure?

That would be #3230